resolver/hosts_file_resolver.go

```diff
@@ -211,6 +216,17 @@ func (r *HostsFileResolver) parseHostsFile() error {
 	return nil
 }
 
+func isLoopbackAddr(addr net.IP) bool {
+	_, loopback4, _ := net.ParseCIDR("127.0.0.0/8")
+	loopback6 := net.ParseIP("::1")
```

Comment:
I think these two vars should be moved out of the function into the file's top level to avoid parsing on every check.

Reply:
Of course, makes sense.

Thanks for the feedback.